docs/ref/request-response.txt

```diff
@@ -38,7 +38,7 @@ All attributes should be considered read-only, unless stated otherwise.
 
 .. attribute:: HttpRequest.body
 
-    The raw HTTP request body as a byte string. This is useful for processing
+    The raw HTTP request body as a bytestring. This is useful for processing
```

Comment:
OK, should this not be just `bytes`, (a `bytestring` being the particular representation of it... `b'some bytes here'`)?

Reply:
Not sure :thinking: It seems that in our documentation `bytes` is mainly used as a unit of size.

Reply:
No, I'm not sure either. It's just that I think we're might be doing it wrong...

```
>>> a_bytestring = b'some content'
>>> type(a_bytestring)
bytes
```

So e.g. the change to 1.7.1.txt, the type returned is `bytes`, which yes we see as a _bytestring_ (with the `b''`) when printed... (and so on for several of the changes here.) 

Happy to not make a thing of it just now. 🙂 (It just came up from our discussion earlier and reviewing here.)

Reply:
Changes from first commit 2eaac7553442baf0d1aafba43b89a583f19ac9cf I treat as a typo. IMO we can merge both commits, or only second if you prefer.

Reply:
Yeah, fine. Agreed it's not `byte strings` 🙂